### DISCUSSION TITLE
Automatic instead of manual execution?

### DISCUSSION THREAD

**Discussion by johnlx86** [lukasmoldon/astats-bot#5] [Q&A] [answered]
Currently I am starting the bot manually. So I wonder what the simplest solution would be for realizing an automatic execution at certain times? Is this possible with the current version of the bot?

**Comment by lukasmoldon** [comment, marked as answer]
For Windows OS you can start the bot using [Windows Task Scheduler](https://en.wikipedia.org/wiki/Windows_Task_Scheduler). 

Just open the "Task Scheduler Library" within the TWS application on the left. 
1. Click on "Action" at the top and create a new simple task. 
2. Enter a name for the task.
3. Click "Actions" and "New", the action should be "Start a program", path of the batchfile should be pasted into "Program/script" and path of the folder containing the batchfile should be pasted in "Start in (optional)".
4. Click "Triggers" and create at least one trigger (e.g. start when system starts or start at specific time).
5. Make sure to save everything before closing TWS.

**Reply by johnlx86** [reply]
Thanks, I will try that!
